package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "lbh-frontend-react",
+  "version": "0.0.1",
```

Comment:
Also update the `package-lock.json` (you can run `npm install` to automate this).